docs/en/deployment.md

```diff
@@ -55,6 +60,39 @@ This automated AWS CloudFormation template deploys the solution in the AWS Cloud
 
 You can view the status of the stack in the AWS CloudFormation Console in the **Status** column. You should receive a CREATE_COMPLETE status in approximately 10 minutes.
 
+### Deploy solution from source code
+
+Deploy solution to your AWS account from CDK.
+#### Prerequisites
+
+* Make sure you have AWS CLI and AWS CDK install in your workspace
+    
+    > You can follow this doc [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) to install AWS CLI.
+   
+    > You can follow this document [CDK Getting Started](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_prerequisites) to install and bootstrap CDK
+
+
+* Make sure you have docker running in your workspace
+
+    > You can follow this document [Docker Install](https://docs.docker.com/engine/install/) to install docker.
+
+
+
+#### Deploy solution from CDK
+ 
+   * Set environment variable `QUICKSIGHT_USER_NAME` to your QuickSight user.
+```sh
+    
+    export QUICKSIGHT_USER_NAME=<your QuickSight user>
+
+```
+
+   * Run below command to deploy the solution.
+```sh
+    cd source
+    npm run deploy
```

Comment:
explicit given the quicksight user in command

Reply:
changed